src/main/java/com/find/doongji/member/payload/request/SignUpRequest.java

```diff
@@ -26,4 +27,6 @@ public class SignUpRequest {
     private String confirmPassword;
 
     private String name;
+
+    private Role role;
```

Comment:
In my DDL for this specific table, I set the default value for Role as ROLE_USER in case no value is decided for the column. 
Do you think it is still necessary to add the validation in the backend?